backend/bitboxbase/handlers/handlers.go

```diff
@@ -49,13 +66,30 @@ func NewHandlers(
 ) *Handlers {
 	handlers := &Handlers{log: log.WithField("bitboxbase", "base")}
 
+	handleFunc("/status", handlers.getStatusHandler).Methods("GET")
+	handleFunc("/channel-hash", handlers.getChannelHashHandler).Methods("GET")
 	handleFunc("/middlewareinfo", handlers.getMiddlewareInfoHandler).Methods("GET")
 	handleFunc("/verificationprogress", handlers.getVerificationProgressHandler).Methods("GET")
-	handleFunc("/connect-electrum", handlers.postConnectElectrumHandler).Methods("POST")
-	handleFunc("/channel-hash", handlers.getChannelHashHandler).Methods("GET")
-	handleFunc("/status", handlers.getStatusHandler).Methods("GET")
+	handleFunc("/backupsysconfig", handlers.postBackupSysconfigHandler).Methods("POST")
+	handleFunc("/backuphsmsecret", handlers.postBackupHSMSecretHandler).Methods("POST")
+	handleFunc("/restoresysconfig", handlers.postRestoreSysconfigHandler).Methods("POST")
+	handleFunc("/restorehsmsecret", handlers.postRestoreHSMSecretHandler).Methods("POST")
+	handleFunc("/resyncbitcoin", handlers.postResyncBitcoinHandler).Methods("POST")
+	handleFunc("/reindexbitcoin", handlers.postReindexBitcoinHandler).Methods("POST")
+	handleFunc("/userauthenticate", handlers.postUserAuthenticate).Methods("POST")
+	handleFunc("/userchangepassword", handlers.postUserChangePassword).Methods("POST")
+	handleFunc("/sethostname", handlers.postSetHostname).Methods("POST")
 	handleFunc("/disconnect", handlers.postDisconnectBaseHandler).Methods("POST")
-	handleFunc("/syncoption", handlers.postSyncOptionHandler).Methods("POST")
+	handleFunc("/connect-electrum", handlers.postConnectElectrumHandler).Methods("POST")
+	handleFunc("/enable-tor", handlers.postEnableTorHandler).Methods("POST")
+	handleFunc("/enable-tor-middleware", handlers.postEnableTorMiddlewareHandler).Methods("POST")
+	handleFunc("/enable-tor-electrs", handlers.postEnableTorElectrsHandler).Methods("POST")
+	handleFunc("/enable-tor-ssh", handlers.postEnableTorSSHHandler).Methods("POST")
+	handleFunc("/enable-clearnet-ibd", handlers.postEnableClearnetIBDHandler).Methods("POST")
+	handleFunc("/enable-root-login", handlers.postEnableRootLoginHandler).Methods("POST")
+	handleFunc("/set-root-password", handlers.postSetRootPasswordHandler).Methods("POST")
+	handleFunc("/shutdown-base", handlers.postShutdownBaseHandler).Methods("POST")
+	handleFunc("/reboot-base", handlers.postRebootBaseHandler).Methods("POST")
```

Comment:
Should decide on one naming convention for Endpoints. Currently it's e.g. `sethostname` and `set-root-password`

Reply:
I think this is fine as `hostname` is normally written as one word

Reply:
Sorry if that wasn't clear: Please take a look at the code. Some endpoints are separated by a hyphen, some are not. `sethostname` and `set-root-password` are just examples.

Reply:
Sorry, that was my bad, didn't read that carefully enough. Then let's go with hyphens between words to keep it consistent with bb02.

Reply:
Changed in https://github.com/digitalbitbox/bitbox-wallet-app/pull/578